lib/DropdownMenu/styles.scss

```diff
@@ -60,7 +60,7 @@ $dropdown-link-padding: 7px 15px !default;
   cursor: pointer;
   color: #6f7c82;
   font-weight: $typo-weight-roman;
-	font-size: $icon-size;
+	font-size: $typo-size-slight;
```

Comment:
Looks like we have some strange indent/spacing issue going off in this file. Can we fix that?